universal-application-tool-0.0.1/app/models/LifecycleStage.java

```diff
@@ -3,6 +3,12 @@
 import io.ebean.annotation.DbEnumType;
 import io.ebean.annotation.DbEnumValue;
 
+/**
+ * An EBean mapped class that represents a stage in a model's lifecycle.
```

Comment:
Nit: this should be a plain enum, not a ebean mapped class.

Reply:
thanks!